src/python/dxpy/scripts/dx.py

```diff
@@ -5634,6 +5802,112 @@ def register_parser(parser, subparsers_action=None, categories=('other', ), add_
+
+parser_archive.add_argument(
+    'path', 
+    help=fill('May refer to a single folder or specify up to 1000 files inside a project.',width_adjustment=-24),
+    default=[], nargs='+').completer = DXPathCompleter() 
+
+parser_archive_output = parser_archive.add_argument_group(title='Output', description='If -q option is not specified, prints "Tagged <count> file(s) for archival"')
+
+parser_archive.set_defaults(func=archive, request_mode = "archival")  
+register_parser(parser_archive, categories='fs')
+
+#####################################
+# unarchive
+#####################################
+
+parser_unarchive = subparsers.add_parser(
+    'unarchive', 
+    help='Requests for the specified set files or for the files in a single specified folder in one project to be unarchived on the platform.',    
+    description=
+'''
+Requests for {} or for the files in {} in {} to be unarchived on the platform.
+The requested copy will eventually be transitioned over to the live state while all other copies will move over to the archival state.
+'''.format(BOLD('a specified set files'), BOLD('a single specified folder'), BOLD('ONE project')) +
+'''
+The input paths should be either 1 folder path or up to 1000 files, and all path(s) need to be in the same project. 
+To specify which project to use, prepend the path or ID of the file/folder with the project ID or name and a colon.
+
+EXAMPLES:
+
+    # unarchive 3 files in project "FirstProj" with project ID project-B0VK6F6gpqG6z7JGkbqQ000Q 
+    $ dx unarchive FirstProj:file-B0XBQFygpqGK8ZPjbk0Q000Q FirstProj:/path/to/file1 project-B0VK6F6gpqG6z7JGkbqQ000Q:/file2
+ 
+    # unarchive 2 files in current project. Specifying file ids saves time by avoiding file name resolution.
+    $ dx select FirstProj
+    $ dx unarchive file-A00000ygpqGK8ZPjbk0Q000Q file-B00000ygpqGK8ZPjbk0Q000Q
+
+    # unarchive all files recursively in project-B0VK6F6gpqG6z7JGkbqQ000Q
+    $ dx unarchive project-B0VK6F6gpqG6z7JGkbqQ000Q:/
+  ''',
+    formatter_class=argparse.RawTextHelpFormatter,
+    parents=[all_arg],
+    prog='dx unarchive')
+
+parser_unarchive.add_argument('--rate', help=fill('The speed at which all files in this request are unarchived.', width_adjustment=-24) + '\n'+ fill('- Azure regions: {Expedited, Standard}', width_adjustment=-24,initial_indent='  ') + '\n'+ 
+fill('- AWS regions: {Expedited, Standard, Bulk}', width_adjustment=-24,initial_indent='  '), choices=["Expedited", "Standard", "Bulk"], default="Standard")
+
+parser_unarchive.add_argument('-q', '--quiet', help='Do not print extra info messages', action='store_true')
+parser_unarchive.add_argument(
+    '-y','--confirm', dest='confirm',
+    help=fill('Do not ask for confirmation. If not specified, dry run will be skipped' , width_adjustment=-24), 
```

Comment:
should say 'Do not ask for confirmation.'

Reply:
Also, should be -y or --yes, and should have 'store_false' as in 
https://github.com/dnanexus/dx-toolkit/blob/master/src/python/dxpy/scripts/dx.py#L5002

```
parser_rmproject.add_argument('-y', '--yes', dest='confirm', help='Do not ask for confirmation', action='store_false')
```

Reply:
similarly for archive

Reply:
Fixed. Thanks for catching that error :)